scripts/build_local_shift.sh

```diff
@@ -4,7 +4,7 @@
 # This is used to start and build services for running e2e tests
 
 set -e
-MINISHIFT_ENABLE_EXPERIMENTAL=y minishift start --extra-clusterup-flags "--service-catalog" \
+MINISHIFT_ENABLE_EXPERIMENTAL=y minishift start --extra-clusterup-flags "--enable='*,service-catalog'" \
```

Comment:
We should probably remove the `service-catalog` flag since we no longer need to test its integration in this repo (see [issue 468](https://github.com/operator-framework/operator-lifecycle-manager/issues/468#issuecomment-422563076)).